app/src/main/res/values/dimens.xml

```diff
@@ -593,4 +593,13 @@
    
   <!-- StoryFragment -->
   <dimen name="story_fragment_padding_bottom">160dp</dimen>
+
+<!--  Revision Card Fragment-->
+  <dimen name="revision_card_fragment_padding_bottom">160dp</dimen>
+  <dimen name="revision_card_fragment_layout_margin_start">28dp</dimen>
+  <dimen name="revision_card_fragment_layout_margin_top_text">36dp</dimen>
+  <dimen name="revision_card_fragment_layout_margin_end">28dp</dimen>
+  <dimen name="revision_card_fragment_layout_margin_top_button">48dp</dimen>
+
+
```

Comment:
@ArchitJain1201 remove this extra line

Reply:
Ma'am How can I remove revision_card_fragment_layout_margin_top_button as its value is being change in port and land mode and even it carry a different value than revision_card_fragment_layout_margin_top_text. ?

Reply:
you don't need to remove it, you wont be able to merge 4 files into 1 as tab layout is different than mobile layout so just merge 2 tab layouts into 1 and 2 mobile into one using the dimens files and adding the required values there

Reply:
changes was made